python/common/python/time.java

```diff
@@ -89,10 +89,26 @@ public time() {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "str(object='') -> str\n"+
```

Comment:
This docstring seems completely unrelated to the one for the ``time.ctime`` function.

Reply:
I agree but this is what python3 returns when `time.ctime().__doc__` .
Is there an alternative ?

Reply:
That's giving you the docstring of the result of calling the function (which is a string).
Try `time.ctime.__doc__`.